### DISCUSSION TITLE
How to add aria-label for the icon we used in the Alert component?

### DISCUSSION THREAD

**Discussion by BoyanLiuu** [cloudscape-design/components#352] [Q&A] [answered]
Hello Friends,
I am trying to solve accessibility violation type : Ensure images provide informative alternative text. 
The specific error : message icons serve a meaningful purpose but are marked as decorative.

The problem happens in the [Alert component](https://cloudscape.design/components/alert/?tabId=playground), So i am trying to solve the violation by adding  aria-label for the warning icon  in the Alert component,
However, it seems like I am not able to do so.
 I look through the documentation and the source code, It seems like we have no ability to add aria-label,
Just want to double check with the Polaris team and want to get a little bit more information about this issue, Thanks for all kinds of help!

I search Alert as keyword and there is no discussion related to the Alert component

**Comment by FlorianDr** [comment, marked as answer]
Hello, 

thank you for reaching out. We are actively working on exposing a property for the aria-label of the icon in `<Alert />`. You can follow the progress on this PR: https://github.com/cloudscape-design/components/pull/347

Thank you,
Florian
